### ISSUE TITLE
Add basic MIT license

### ISSUE THREAD

**Issue by awfulwoman** [unfrl/dug#47]
I'd love to add this (or see this added) to Homebrew, but the lack of any recognised license (#3) stops that happening.

Is there any chance of adding the most simple MIT license? It is the most simple one possible, allowing only redistribution and stating that no warranty is granted.

https://opensource.org/license/MIT

**Comment by MattInternet** [comment]
Yeah that is an annoying limitation. It looks like your PR just deletes the license though instead of replacing it? If I was to change the license would you be willing to package dug for homebrew?

**Comment by awfulwoman** [comment]
ah, sorry about that - looks like I never pushed the correct branch. 😱 Fixed. 

I can certainly package dug for homebrew later this week if this gets merged!

**Comment by MattInternet** [comment]
Excellent! Merged, Ill tag a release too

**Comment by awfulwoman** [comment]
The PR for the homebrew package is awaiting review and approval. https://github.com/Homebrew/homebrew-core/pull/192974

**Comment by awfulwoman** [comment]
Just waiting on the release to be cut so this can be merged into homebrew! https://github.com/unfrl/dug/issues/49
